### DISCUSSION TITLE
Add support for type imports in typescript

### DISCUSSION THREAD

**Discussion by sfabriece** [rome/tools#4531] [Suggestions]
I'm evaluating Rome and I love the ideas behind it. The one feature I'm missing from typescript eslint is the ability to detect imports that are on types and adding type to these.

Is this something you see adding to Rome?

**Comment by Conaclos** [comment]
Which feature do you request? The code action of automatically turning import of types into `import type { X }`/`import { type X }`?

Otherwise, you can use TSC [--verbatimModuleSyntax](https://devblogs.microsoft.com/typescript/announcing-typescript-5-0-beta/#verbatimmodulesyntax) compiler option.
